tests/test_move.py

```diff
@@ -37,6 +38,17 @@ def test_move_fs(self):
             self.assertEqual(dst_file1_info.modified, src_file1_info.modified)
             self.assertEqual(dst_file2_info.modified, src_file2_info.modified)
 
+    def test_move_file(self):
```

Comment:
It looks like this method was written explicitly to test your "optimised" code-path?
Might also be worth adding a version that moves a file from one memfs to another memfs, to test the non-optimised code-path?

Reply:
Just added that 👍